model.js

```diff
@@ -4,4 +4,9 @@ const displayCategories = () => {
   return db.query(`SELECT * FROM categories;`);
 };
 
-module.exports = { displayCategories };
+const displayReviewId = (reviewId) => {
+  return db.query(`SELECT * FROM reviews
+                   WHERE review_id = ${reviewId}`);
```

Comment:
we should be using the `$` syntax paired with a values array to prevent SQL injection